sympy/polys/ring_series.py

```diff
@@ -1638,7 +1642,7 @@ def rs_tanh(p, x, prec):
                 const = R(tanh(c))
             except ValueError:
                 raise DomainError("The given series can't be expanded in this "
-                "domain")
+                    "domain")
```

Comment:
Output by flake8.

> sympy/polys/ring_series.py:1645:21: E128 continuation line under-indented for visual indent

By aligning ", this can be fixed.